### ISSUE TITLE
[Accessibility]: VoiceOver stops reading CodeMirror suggestions while typing

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#43137]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34188

### Problem description

VoiceOver sometimes interrupts reading CodeMirror suggestions in a very odd and confusing way. See video.

https://user-images.githubusercontent.com/206864/196557332-cde364a4-1732-4989-9db4-012c869fc541.mp4

### Expected behavior

The suggestions should be read entirely, or not at all.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @lguychard

**Comment by fkling** [comment]
Mmh, I just tested it on .com and I think it's still the same as before. Can you give it a try just to make sure?

**Comment by limitedmage** [comment]
Yeah I still see the same issue :( 

I think we should timebox the investigation (a few hours) to see if a fix is possible. I couldn't repro this same issue in [CodeMirror's demo instance](https://codemirror.net/try/), so it might be related to the way we are using it. Since you are most familiar with CodeMirror, can you give it a shot?

**Comment by limitedmage** [comment]
Tested this out on Windows with NVDA (couldn't get Narrator to work with CodeMirror) and it works beautifully. So I think this is a bug with VoiceOver. Here's a video:

https://user-images.githubusercontent.com/206864/200647703-643da7e4-4f91-47d4-b3fb-bd32db08916e.mp4
